_inc/idc-notice.js

```diff
@@ -15,21 +15,31 @@
 		fixJetpackConnection();
 	} );
 
+	function disableDopsButtons() {
+		$( '.dops-button' ).prop( 'disabled', true );
+	}
+
+	function enableDopsButtons() {
+		$( '.dops-button' ).prop( 'disabled', false );
```

Comment:
Instead of having this jQuery selector in each function, we should probably add it at the top of the file under `notice` so that we're only doing it once per page load.

Also, we should probably scope the selector a bit more. How about something like this:

`var dopsButton = $( '.jp-idc-notice .dops-button' );`